src/cursor/cur_log.c

```diff
@@ -208,7 +208,7 @@ __curlog_next(WT_CURSOR *cursor)
      */
     if (cl->stepp == NULL || cl->stepp >= cl->stepp_end || !*cl->stepp) {
         cl->txnid = 0;
-        ret = __wt_log_scan(session, cl->next_lsn, WT_LOGSCAN_ONE, __curlog_logrec, cl);
+        ret = __wt_log_scan(session, cl->next_lsn, WT_LOGSCAN_ONE, __curlog_logrec, cl, NULL, NULL);
```

Comment:
The 2nd arg is already a starting LSN. So the new end LSN arg should be added as the 3rd arg and only 1 new arg needs to be added.